rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    book.rentals << self
+    @person = person
+    person.rentals << self
+  end
+
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
- [ ] I suggest that you make `book` and `person` `attr_accessor ` to be able to access and modify them